plone/app/portlets/portlets/navigation.py

```diff
@@ -1,6 +1,7 @@
 # -*- coding: utf-8 -*-
 from Acquisition import aq_inner, aq_base, aq_parent
 from ComputedAttribute import ComputedAttribute
+from plone import api
```

Comment:
@ebrehault I thought  that the decision so far was to *not* allow plone.api in the core... I'm not much against it as long as the circular dependencies are taken care of 😓